mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 minY I
```

Comment:
`getMinY` does not return the value of this field.

Reply:
```suggestion
```
Please also remove this unknown mapping.

Reply:
@modmuss50 If martmists allows project maintainer to edit his pr, please remove this mapping entry and things are good to go.